gio/build.rs

```diff
@@ -1,22 +1,3 @@
-fn main() {
-    manage_docs();
-}
-
-#[cfg(all(
-    any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"),
-    not(all(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"))
-))]
-fn manage_docs() {
-    const PATH: &str = "src";
-    const IGNORES: &[&str] = &["lib.rs", "prelude.rs"];
-    lgpl_docs::purge(PATH, IGNORES);
-    if cfg!(feature = "embed-lgpl-docs") {
-        lgpl_docs::embed(lgpl_docs::Library::Gio, PATH, IGNORES);
-    }
-}
-
-#[cfg(any(
-    all(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"),
-    not(any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"))
-))]
-fn manage_docs() {}
+// This whole build.rs file is required because of the `OUT_DIR` environment variable
+// provided by cargo when a build file is present.
+fn main() {}
```

Comment:
That's not totally descriptive (might want to mention the tests need `OUT_DIR` to store a temporary file, and that environment variable is only defined when the crate has a `build.rs` script), but okay.